projects/exui-common-lib/src/lib/components/hmcts-dialog/hmcts-dialog.component.ts

```diff
@@ -0,0 +1,28 @@
+import {DOCUMENT} from '@angular/common';
+import {Component, EventEmitter, Inject, Input, OnInit, Output} from '@angular/core';
+/*
+* Gov UK Dialog Component
+* Responsible for displaying dialog layout
+* using ng-content to display content from parent
+* */
+@Component({
+  selector: 'lib-dialog',
+  templateUrl: './hmcts-dialog.component.html',
+  styleUrls: ['./hmcts-dialog.component.scss']
+})
+export class HmctsDialogComponent  implements OnInit {
+
+  @Input() public positionTop: string;
+  @Output() public close = new EventEmitter();
+  constructor(@Inject(DOCUMENT) private document: any) {}
+
+  public ngOnInit(): void {
+    const dialogEl = this.document.querySelector(`.gem-c-modal-dialogue__box`);
+    dialogEl.focus();
```

Comment:
As this is a common library I'm not too happy using magic selectors for elements that the implementing app has to use for this to work. Much better would be to have a look at how angular material does the dialog, as that's a much cleaner implementation

Reply:
have got advice from Simon which dialogue to pick up... but i see what you mean.